jenkins/common.groovy

```diff
@@ -222,6 +222,11 @@ def checkForBrokenLinks(String buildLogPath, String envName, Boolean contentOnly
       return;
     }
 
+    // JSONObjects in Groovy are not serializable by default, which is an issue, because
+    // a Jenkinsfile has to be fully serializable for it to be able to pause state.
+    // To get around this,
+    brokenLinks = null
+
```

Comment:
On [this Content Build](http://jenkins.vfs.va.gov/job/builds/job/vets-website-content-vagovprod/1308/console) I noticed this output 

![image](https://user-images.githubusercontent.com/1915775/116612204-44d9ff00-a905-11eb-9cb6-e9e485ac8af7.png)

There shouldn't have been an exception for only 1 broken link. A Google search for `Caused: java.io.NotSerializableException: groovy.json.internal.LazyMap` yielded some interesting Jenkins results, like this one -

> Jenkins has a paradigm where all jobs can be interrupted, paused and resumable through server reboots. To achieve this the pipeline and its data must be fully serializable - IE it needs to be able to save the state of everything. Similarly, it needs to be able to serialize the state of global variables between nodes and sub-jobs in the build, which is what I think is happening for you and I and why it only occurs if you add that additional build step.

The suggestion to fix this is 

> Try unsetting your object var or wrapping it in a method so its scope isn't node global.

I am betting that while the `slackSend` function is running, Jenkins serializes the state of the job, and that's where it hits an exception. I think unsetting `brokenLinks` before `slackSend` will fix it.

https://stackoverflow.com/questions/37864542/jenkins-pipeline-notserializableexception-groovy-json-internal-lazymap